lib/ansible/modules/cloud/vmware/vmware_datastore_maintenancemode.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_datastore_maintenancemode
+short_description: Place a datastore into maintenance mode
+description:
+    - This module can be used for to manage maintenance mode of datastore.
```

Comment:
`This module can be used to manage maintenance mode of a datastore` (small change: remove 'for', add 'a')

Reply:
OK.